tests/library/browsercontext-fetch.spec.ts

```diff
@@ -409,20 +409,83 @@ it('should return error with wrong credentials', async ({ context, server }) =>
   expect(response2.status()).toBe(401);
 });
 
-for (const method of ['delete', 'get', 'head', 'patch', 'post', 'put']) {
```

Comment:
Yeah, it's unfortunate that our lint doesn't catch errors like this when `context.request[method]` syntax is used.